back-end/helpers/intent_analyzer.js

```diff
@@ -11,7 +11,7 @@ export default class IntentAnalyzer{
   formatModel(model) {//Formats the mongoose model into desired array for IntentLister to use
     let percentageMap;
     if (model.total_children == 0) {
-      percentageMap = {"No intents found.": 0};
+      percentageMap = {"[END OF CONVERSATION]": 0};
```

Comment:
I was thinking I can add this to the formatter so that there is an "End of the conversation" children and it has its own percentage. Because according to the feedbacks I got, people don't know how to flag questions. So maybe if we have that end of conversation we will have more info and it will be basically our "check for quitting" icebox idea. And we will add some extra conversation to our database to have conversations that end early? Just an idea we can talk about this later. But if we do that there will be no situation where a question doesn't have children